krm-functions/sig-cli/render-helm-chart/third_party/README.md

```diff
@@ -0,0 +1 @@
+This package is forked from `sigs.k8s.io/kustomize/api/` with version `api/0.8.11`.
```

Comment:
Can you remind me why we did this? Do we still need to do it now?

Reply:
This function uses the same code as kustomize's `HelmChartInflator` plugin so that we do not need to rewrite it. It is not possible to remove this fork without having to rewrite the function, because the kustomize plugin is in an internal package.

Reply:
Got it